spec/hydra/works/models/concerns/generic_file/characterization_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+# Mocking out the Hydra::Works::GenericFile sufficiently to add original_file
```

Comment:
I have a slight preference to see this code (lines 3-22) put in a helper in `spec/support/` and `require`d at the top of this file.